src/tools/dev/scripts/run-build-visit

```diff
@@ -11,11 +11,11 @@
 #    Valid machine names:
 #       kickit_mesagl (B-Div, Linux, x86_64, mesagl),
 #       kickit_opengl (B-Div, Linux, x86_64, opengl),
-#       poodle (LC, Linux, x86_64, toss4),
-#       lassen (LC, Linux, power9),
-#       rzgenie (LC, Linux, x86_64, toss3),
-#       rztopaz (LC, Linux, x86_64, toss4),
-#       trinity (LANL, x86_64)
+#       poodle    (LC, Linux, x86_64, toss4),
+#       lassen    (LC, Linux, power9),
+#       rzwhippet (LC, Linux, x86_64, toss4),
```

Comment:
Do we actually need rzwhippet?
Does the rztopaz path use the same build steps as rzwhippet?

Reply:
rzwhippet is a much faster machine, so it will be nice to have even if we just use for development